doc/user-guides/secure-protect-connect-single-multi-cluster.md

```diff
@@ -314,7 +323,7 @@ EOF
 To check your rate limits have been accepted, enter the following command:
 
 ```bash
-kubectl get ratelimitpolicy ${gatewayName}-rlp -n ${gatewayNS} -o=jsonpath='{.status.conditions[?(@.type=="Accepted")].message}'
+kubectl get ratelimitpolicy ${GATEWAY_NAME}-rlp -n ${GATEWAY_NS} -o=jsonpath='{.status.conditions[?(@.type=="Accepted")].message}'
 ```
```

Comment:
Can be a follow on, but when we wrote this initially we didn't have enforced conditions on our policies. I think it would be good to show the enforced status and call out waiting for that before setting up DNSPolicy to bring traffic to your gateway